.moai/specs/SPEC-PHASE-FRONTMATTER-OWNER-001/progress.md

```
@@ -0,0 +1,428 @@
+
+```
+$ grep -c 'FALSIFICATION' internal/spec/lint_phase.go internal/spec/lint.go
+internal/spec/lint_phase.go:0
+internal/spec/lint.go:0
+```
+
+변형 잔재 0 — 세 반증의 되돌림이 완전하다.
+
+### M5 — 9개 정정 (AC-PFO-012 / AC-PFO-013)
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)